src/main/java/net/dv8tion/jda/internal/handle/ChannelUpdateHandler.java

```diff
@@ -336,20 +336,20 @@ private void addChanged(List<IPermissionHolder> changed, Guild guild, long id)
             changed.add(holder);
     }
 
-    private boolean handlePermissionOverride(PermissionOverride currentOverride, DataObject override, long id, AbstractChannelImpl<?,?> channel)
+    private boolean handlePermissionOverride(PermissionOverride currentOverride, DataObject override, long overrideId, AbstractChannelImpl<?,?> channel)
```

Comment:
This is the `permissionHolderId`, not an `id` specific to the override entity.

Reply:
The override id is the perm holder id in the api.

Reply:
Ah, the actual param is called override id? 
Okay.